src/main/java/io/fabric8/maven/docker/BuildMojo.java

```diff
@@ -105,7 +105,9 @@ private void proceedWithDockerBuild(ServiceHub hub, BuildService.BuildContext bu
             } else {
                 buildService.buildImage(imageConfig, pullManager, buildContext, buildArchiveFile);
                 if (!skipTag) {
-                    buildService.tagImage(imageConfig);
+                    if (!imageConfig.getBuildConfiguration().skipTag()) {
```

Comment:
Can this `if` statement merged with the one above?

Reply:
```
if (!skipTag && !imageConfig.getBuildConfiguration().skipTag()) {
   buildService.tagImage(imageConfig);
}
```

It looks like I should use `&&`, is it correct?

Reply:
Yes, looks correct to me.